app/controllers/api/v1/material_batches_controller.rb

```diff
@@ -1,89 +1,45 @@
 class Api::V1::MaterialBatchesController < Api::V1::ApplicationController
-  before_action :set_material_batch, only: [:show, :update, :destroy]
+  before_action :set_material_batch, only: [:show, :update]
 
   # GET /material_batches
   def index
     @material_batches = MaterialBatch.all
 
-    render json: @material_batches, include: [:materials, "materials.material_type", "materials.metadata"]
+    render json: @material_batches, include: includes
   end
 
   # GET /material_batches/1
   def show
-    render json: @material_batch, include: [:materials, "materials.material_type", "materials.metadata"]
+    render json: @material_batch, include: includes
```

Comment:
If the material_batch is going to be serialized to json in the same way (with that list of includes), maybe we could move this to the model. Maybe redefining as_json for the model? Or creating a serializer, like they say in https://robots.thoughtbot.com/better-serialization-less-as-json?